addons/kube-state-metrics/template.cue

```diff
@@ -0,0 +1,36 @@
+output: {
+    apiVersion: "core.oam.dev/v1beta1"
+    kind: "Application"
+    metadata: {
+        name: parameter.name
```

Comment:
This metadata actually have no effect on the addon.

Reply:
OK